src/CoreWlan/Enums.cs

```diff
@@ -116,4 +116,20 @@ public enum CWKeychainDomain : nuint_compat_int {
 		User = 1,
 		System = 2,
 	}
+
+	[Native]
+	public enum CWEventType : nuint_compat_int {
+		None = 0,
+		PowerDidChange = 1,
+		SSIDDidChange = 2,
+		BSSIDDidChange = 3,
+		CountryCodeDidChange = 4,
+		LinkDidChange = 5,
+		LinkQualityDidChange = 6,
+		ModeDidChange = 7,
+		ScanCacheUpdated = 8,
+		VirtualInterfaceStateChanged = 9,
+		RangingReportEvent = 10,
+		Unknown = int.MaxValue
```

Comment:
@spouliot Since this is an nint, this'll be wrong on 64-bit right? What's the right way to bind:

    CWEventTypeUnknown                  = NSIntegerMax,